webpage/_data/navigation.yml

```diff
@@ -13,3 +13,9 @@ site_nav:
     children:
       - title: "Installing the IOOS conda environment"
         url: /other_resources
+      - title: "Australian ocean data network portal user guide"
```

Comment:
Done in https://github.com/ioos/notebooks_demos/pull/166/commits/07dd6e39a804ec1b95c1fd9f5d4e875a1ed0a020